apps/sms/tools/create_bridge_lib.sh

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
right, I update the content but forgot to update name. 'update_libs.sh' sounds good to me.